service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinNotificationEmailService.kt

```diff
@@ -90,6 +90,14 @@ class BulletinNotificationEmailService(
         }
     }
 
+    private fun getLanguage(languageStr: String?): Language {
+        return when (languageStr) {
+            "sv", "SV" -> Language.sv
+            "en", "EN" -> Language.en
```

Comment:
You could do `language.toLowerCase()` so you wouldn't have to always give the matches in different cases.